README.md

```diff
@@ -79,14 +79,14 @@ the COOL environment.
 
 | Name | Version |
 |------|---------|
-| aws | ~> 3.38 |
-| aws.dns\_sharedservices | ~> 3.38 |
-| aws.organizationsreadonly | ~> 3.38 |
-| aws.provisionassessment | ~> 3.38 |
-| aws.provisionparameterstorereadrole | ~> 3.38 |
-| aws.provisionsharedservices | ~> 3.38 |
-| cloudinit | ~> 2.0 |
-| null | ~> 3.0 |
+| aws | 3.59.0 |
+| aws.dns\_sharedservices | 3.59.0 |
+| aws.organizationsreadonly | 3.59.0 |
+| aws.provisionassessment | 3.59.0 |
+| aws.provisionparameterstorereadrole | 3.59.0 |
+| aws.provisionsharedservices | 3.59.0 |
+| cloudinit | 2.2.0 |
+| null | 3.1.0 |
```

Comment:
I don't think we want to change these versions in the documentation until we implement provider version locking.

Reply:
That's what the latest version of `terraform-docs` gave me.  What should I change to make the latest version of `terraform-docs` behave?

Reply:
You need to add `--lockfile=false` to the list of arguments and that will preserve the versions we specify in `versions.tf`.

Reply:
Please see commit 21b49c9.